tasks/broad/InternalArraysTasks.wdl

```diff
@@ -330,3 +330,39 @@ task UpdateChipWellBarcodeIndex {
     preemptible: preemptible_tries
   }
 }
+
+task GetNextArraysQcAnalysisVersionNumber {
+  input {
+    String chip_well_barcode
+    Int preemptible_tries
+    Array[String] authentication
+    String service_account_filename
+  }
+
+  meta {
+    volatile: true
+  }
+
+  command <<<
+    set -eo pipefail
+
+    AUTH=~{write_lines(authentication)} && chmod +x $AUTH && $AUTH
+    export GOOGLE_APPLICATION_CREDENTIALS=/cromwell_root/~{service_account_filename}
+
+    java -Xms2g -Dpicard.useLegacyParser=false -jar /usr/gitc/picard-private.jar \
+      GetNextArraysQcAnalysisVersionNumber \
+        --CHIP_WELL_BARCODE ~{chip_well_barcode} \
+        --DB_USERNAME_FILE cloudsql.db_user.txt \
+        --DB_PASSWORD_FILE cloudsql.db_password.txt \
+        --DB_JDBC_FILE cloudsql.db_jdbc.txt
+  >>>
+
+  runtime {
+    docker: "us.gcr.io/broad-arrays-prod/arrays-picard-private:4.0.10-1631039849"
```

Comment:
are you not changing the other tasks that previously used `us.gcr.io/broad-arrays-prod/arrays-picard-private:4.0.10-1602016912` ?

Reply:
Good point.